src/libraries/Microsoft.PowerFx.Interpreter/Functions/LibraryText.cs

```diff
@@ -746,7 +786,7 @@ private static FormulaValue Substitute(IRContext irContext, FormulaValue[] args)
                 }
             }
 
-            return new StringValue(irContext, sourceValue);
+            return FormulaValue.New(sourceValue);
```

Comment:
>return FormulaValue.New(sourceValue); [](http://example.com/codeflow?start=12&length=37)

We should keep the irContext in the values that we use - this may help if this new value is used in an expression that generates an error

Reply:
The IR source spans are wrong and we can't meaningfully use them.

Reply:
Re-added to stay consistent. We can remove later.